tests/Unit/Facade.Tests/PurchaseOrdersReportServiceTests/ContextBase.cs

```diff
@@ -1,4 +1,4 @@
-﻿namespace Linn.Purchasing.Facade.Tests.ResourceBuilderTests.OrdersBySupplierReportServiceTests
+﻿namespace Linn.Purchasing.Facade.Tests.ResourceBuilderTests.PurchaseOrdersReportServiceTests
```

Comment:
Is this a ResourceBuilder test? Looks like the Sut in this context is the PurchaseOrderReportFacadeService

Reply:
(just the namespace that looks wrong I mean, not the actual test)

Reply:
was just facade service tests yeah, have amended the namespace! Good spot. I guess these will also indirectly test the resource builder but just took the style of tests from stores

Reply:
They do in this case, but they probably shouldn't since these are unit tests. I'd advocate for seperate unit tests for the builders, and then mocking them out in here